decidim-participatory_processes/app/helpers/decidim/participatory_processes/participatory_process_helper.rb

```diff
@@ -64,7 +64,16 @@ def process_nav_items(participatory_space)
                      .published.or(Decidim::Component.where(id: try(:current_component)))
                      .where(visible: true)
 
-        components.map do |component|
+        [
+          *(if participatory_space.members_public_page?
+              [{
+                name: t("member_menu_item", scope: "layouts.decidim.participatory_process_navigation"),
+                url: decidim_participatory_processes.participatory_process_participatory_space_private_users_path(participatory_space),
+                active: is_active_link?(decidim_participatory_processes.participatory_process_participatory_space_private_users_path(participatory_space), :inclusive)
+              }]
+            end
+           )
+        ] + components.map do |component|
```

Comment:
do we need a `.compact` after the `.end` of this `.map`?

Reply:
Done